src/code/client.coffee

```diff
@@ -39,7 +41,16 @@ class CloudFileManagerClient
 
     # filter for available providers
     allProviders = {}
-    for Provider in [ReadOnlyProvider, LocalStorageProvider, GoogleDriveProvider, LaraProvider, DocumentStoreProvider, LocalFileProvider]
+    providerList = [
```

Comment:
I think you should add a note here about the order of providers being important when it searches for an auto provider in `autoprovider()` as it returns the first one that marks the capability as `auto`.  It might help in the future if other providers are added that mark the same capabilities as `auto`.

Reply:
Done. It practice I don't think it will matter much as the interest in auto-providers mostly comes from clients who are configuring the CFM to only have a single provider, but the comment makes sense in any case.